core/src/components/toolbar/test/colors/index.html

```diff
@@ -36,147 +34,140 @@
-              <ion-button fill="outline">Outline</ion-button>
-              <ion-button>
-                <ion-icon slot="icon-only" ios="ellipsis-horizontal" md="ellipsis-vertical"></ion-icon>
-              </ion-button>
+              <ion-button fill="outline" class="ion-focused">Outline</ion-button>
+              <ion-button fill="outline" class="ion-activated">Outline</ion-button>
             </ion-buttons>
             <ion-title>Light</ion-title>
           </ion-toolbar>
 
           <ion-toolbar color="medium">
             <ion-buttons slot="start">
-              <ion-button fill="solid" color="danger">
-                <ion-icon slot="icon-only" name="add"></ion-icon>
-              </ion-button>
-              <ion-button fill="solid">Solid</ion-button>
+              <ion-button fill="clear">Clear</ion-button>
             </ion-buttons>
             <ion-buttons slot="primary">
-              <ion-button fill="clear">Clear</ion-button>
-              <ion-button>
-                <ion-icon slot="icon-only" ios="ellipsis-horizontal" md="ellipsis-vertical"></ion-icon>
-              </ion-button>
+              <ion-button fill="clear" class="ion-focused">Clear</ion-button>
+              <ion-button fill="clear" class="ion-activated">Clear</ion-button>
             </ion-buttons>
             <ion-title>Medium</ion-title>
           </ion-toolbar>
 
           <ion-toolbar color="dark">
             <ion-buttons slot="start">
-              <ion-button fill="solid" color="danger">
-                <ion-icon slot="icon-only" name="add"></ion-icon>
-              </ion-button>
               <ion-button fill="solid">Solid</ion-button>
             </ion-buttons>
             <ion-buttons slot="primary">
-              <ion-button fill="clear">Clear</ion-button>
-              <ion-button>
-                <ion-icon slot="icon-only" ios="ellipsis-horizontal" md="ellipsis-vertical"></ion-icon>
-              </ion-button>
+              <ion-button fill="solid" class="ion-focused">Solid</ion-button>
+              <ion-button fill="solid" class="ion-activated">Solid</ion-button>
             </ion-buttons>
             <ion-title>Dark</ion-title>
           </ion-toolbar>
 
+          <hr />
+
           <ion-toolbar class="custom-toolbar">
             <ion-buttons slot="start">
               <ion-button fill="solid" color="danger">
                 <ion-icon slot="icon-only" name="add"></ion-icon>
               </ion-button>
-              <ion-button fill="solid">Solid</ion-button>
             </ion-buttons>
             <ion-buttons slot="primary">
-              <ion-button fill="clear">Clear</ion-button>
               <ion-button>
                 <ion-icon slot="icon-only" ios="ellipsis-horizontal" md="ellipsis-vertical"></ion-icon>
               </ion-button>
             </ion-buttons>
             <ion-title>Custom</ion-title>
           </ion-toolbar>
+
+          <ion-toolbar class="custom-toolbar">
```

Comment:
Some of the screenshots are getting cut off down here, examples:
- `core/src/components/toolbar/test/colors/toolbar.e2e.ts-snapshots/toolbar-colors-md-ltr-Mobile-Chrome-linux.png`
- `core/src/components/toolbar/test/colors/toolbar.e2e.ts-snapshots/toolbar-colors-md-ltr-Mobile-Firefox-linux.png`
- `core/src/components/toolbar/test/colors/toolbar.e2e.ts-snapshots/toolbar-colors-md-ltr-Mobile-Safari-linux.png`

Reply:
Looking into fixing this!

Reply:
Should be fixed now.